README.md

```diff
@@ -119,7 +119,12 @@ defer cur.Close(ctx)
 for cur.Next(ctx) {
    var result bson.D
    err := cur.Decode(&result)
-   if err != nil { log.Fatal(err) }
+    if err == mongo.ErrNoDocuments {
```

Comment:
This is the result of the `Find` on line 116. Should this check instead be on the `FindOne` on line 139 below?

Reply:
Thanks for pointing it out.